WISE-Community/WISE
# Setting StudentDataType default to Responses in Summary Component

**Steps to Test**

1. Log in as a teacher
2. Create a new project or open an existing project
3. Create a step with a Multiple Choice component
4. Create another step with a Summary component
5. Author the Summary component to point to the Multiple Choice component you previously made
6. The "Student Data Type" should now automatically be set to "Responses"  instead of "Scores"

Concerning changes to package files

- Due to dependency changes, the packages below were updated upon running `npm install `

![image](https://user-images.githubusercontent.com/14948397/68510058-5d44ee00-0227-11ea-888c-485dae47f59c.png)

Closes #2013
